### PR TITLE
Completed French translation & hallowed infested blocks change

### PR DESCRIPTION
**Pull request by YanisBft**
Infested blocks from the mod now spawn ~~bats~~ endermites instead of silverfishes

### CODE REVIEW

**Comment by Shnupbups** [comment]
Gonna have to say no to the bats thing. Maybe we'll do something funky with the Infested Blocks later, but for now they should stay Silverfish IMO (or maybe Endermites?)

Actually, if you could make it Endermites, I'll be down for that

**Comment by YanisBft** [comment]
Done

**Review by Shnupbups — APPROVED** [review]

File: src/main/java/com/fabriccommunity/thehallow/block/HallowedInfestedBlock.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+	public HallowedInfestedBlock(Block block, Settings settings) {
+		super(block, settings);
+	}
+	
+	@Override
+	public void onStacksDropped(BlockState state, World world, BlockPos pos, ItemStack stack) {
+		if (!world.isClient && world.getGameRules().getBoolean(GameRules.DO_TILE_DROPS) && EnchantmentHelper.getLevel(Enchantments.SILK_TOUCH, stack) == 0) {
+			EndermiteEntity endermite = (EndermiteEntity) EntityType.ENDERMITE.create(world);
+			endermite.setPositionAndAngles(pos.getX() + 0.5D, pos.getY(), pos.getZ() + 0.5D, 0.0F, 0.0F);
```
Comment (liach) [inline src/main/java/com/fabriccommunity/thehallow/block/HallowedInfestedBlock.java:25]: If we have a way to obtain the orientation of the player who broke the block we can consider making the mite face the player

**Review by liach — APPROVED** [review]

File: src/main/java/com/fabriccommunity/thehallow/block/HallowedInfestedBlock.java (hunk 1 above)
Comment (Shnupbups) [reply src/main/java/com/fabriccommunity/thehallow/block/HallowedInfestedBlock.java:25]: Eh, I'd prefer to keep the functionality identical to the vanilla Infested blocks

**Review by Juuxel — APPROVED** [review]

**Review by LemmaEOF — APPROVED** [review]

**Pull request merged**